pkg/controller/defaults/testdata/volumeclass/volume-class-defaults-set/input.yaml

```diff
@@ -0,0 +1,26 @@
+kind: AppInstance
+apiVersion: internal.acorn.io/v1
+metadata:
+  generation: 2
+  name: app-name
+  namespace: app-namespace
+  uid: 1234567890abcdef
+spec:
+  image: test
+status:
+  observedGeneration: 1
+  namespace: app-created-namespace
+  appImage:
+    id: test
+  appSpec:
+    containers:
+      container-name:
+        image: "image-name"
+        dirs:
+          "/var/tmp":
+            volume: foo
+  defaults:
+    volumes:
+      foo:
+        class: test-volume-class
+        size: 4Gi
```

Comment:
In this particular test instance, we expect this to remain 4Gi after going through a reconciliation.